### DISCUSSION TITLE
Telemetry in Astro

### DISCUSSION THREAD

**Discussion by morriq** [withastro/roadmap#989] [Proposal]
### Body

Astro should respect `DO_NOT_TRACK` env variable. More about [it](https://consoledonottrack.com/)

**Comment by Fryuni** [comment]
I agree with Gatsby and Homebrew takes on this, there is no standard for this. The same point could be made for a `NO_TELEMETRY`, `DIABLE_TELEMETRY`, `FORCE_LOCAL_FIRST` or any other environment variable for the same purpose. There is no reason to accept this one specifically.

Even the author of the project seems to have abandoned it. The [repository](https://github.com/sneak/consoledonottrack.com) that the page points to was deleted, yet the website wasn't updated. If you go out of your way to find out what happened, you'll find that the author moved to https://git.eeqj.de/sneak/consoledonottrack.com

The project has a single commit since 2019, one earlier this year, which adds a statement saying that the project was a mistake even for its own goal: https://git.eeqj.de/sneak/consoledonottrack.com/src/branch/master/index.markdown#update-2024
